iphone/Maps/UI/BottomMenu/TabBar/BottomTabBarViewController.swift

```diff
@@ -56,9 +56,14 @@ class BottomTabBarViewController: UIViewController {
   @IBAction func onSearchButtonPressed(_ sender: Any) {
     presenter.onSearchButtonPressed()
   }
-
+  
   @IBAction func onHelpButtonPressed(_ sender: Any) {
-    presenter.onHelpButtonPressed()
+    guard helpBadge.isHidden else {
```

Comment:
Why guard/return is better than a simple if/else?

Reply:
Just trying to avoid additional scope for this super rare case with badge is visible. 
Fixed to if/else!